packages/client/src/declarations/index.ts

```diff
@@ -1426,6 +1450,13 @@ export const declarationsReducer: LoopReducer<IDeclarationsState, Action> = (
       )
     }
     case DELETE_DECLARATION_SUCCESS:
+      const declarationToDelete = state.declarations.find(
+        (declaration) => declaration.id === action.payload
+      )
+      const declarationMinioUrls =
+        getMinioUrlsFromDeclaration(declarationToDelete)
+
+      postMinioUrlsToServiceWorker(declarationMinioUrls)
```

Comment:
Ideally we would like to dispatch the `DELETE_DECLARATION_SUCCESS` _**after**_ we have cleared the cache for all the minio urls